packages/gestalt/src/IconButtonFloating.js

```diff
@@ -67,13 +67,13 @@ const IconButtonFloatingWithForwardRef: React$AbstractComponent<Props, unionRefs
   ref,
 ): Node {
   return (
-    <Box borderStyle="shadow" rounding="circle">
+    <Box borderStyle="shadow" rounding="circle" color="default">
       <IconButton
         accessibilityControls={accessibilityControls}
```

Comment:
typo: in the description for `selected`: "…the associated Dropdown is open or _closed_."